### ISSUE TITLE
Remove test performance report during training

### ISSUE THREAD

**Issue by CunliangGeng** [DeepRank/deeprank#121]
**Sonja**:  just looking at the plot on the DeepFace problem reminded me of something that we forgot to take care of. When we train, in every epoch we plot the training/validation/test performances. However, we all know that this is wrong, as one can use the test performances to decide how to proceed with training. What we really want is to keep the performances on the test set invisible until the very last moment, when the final neural network model is chosen. 
So we need to update DeepRank appropriately.

**Comment by CunliangGeng** [comment]
I have checked the code and prefer not changing the code currently. 
To not report the test during training, you just need to set only the ratio between training data and validation data or just None for the parameter `divide_trainset` in function `model.train`. Then it won't calculate and report the test performance. You can always use function `model.test` to get the test performance after selecting proper models based on validation results.

Close the issue now.
